manage-data/data-store/data-streams/tsds-ingest-otlp.md

```diff
@@ -95,6 +95,29 @@ By default, metrics are ingested into the `metrics-generic.otel-default` data st
 
 The target data stream name is constructed as `metrics-${data_stream.dataset}.otel-${data_stream.namespace}`.
 
+## Configure histogram handling
+```{applies_to}
+stack: preview 9.3
+```
+
+You can configure how OTLP histograms are mapped using the `xpack.otel_data.histogram_field_type` cluster setting. Valid values are:
```

Comment:
this setting should also be documented in the reference (maybe [here](https://www.elastic.co/docs/reference/elasticsearch/configuration-reference/index-management-settings)?)